pkg/api/api.proto

```diff
@@ -253,6 +253,13 @@ message UpstreamNotDeployed {
 message Environment {
 
   message Application {
+    message Environment {
+      message DeploymentMetaData {
+        string deployAuthor = 1;
+        string deployTime = 2;
```

Comment:
```suggestion
        int64 deployAtSecondsUtc = 2;
```
It should be clear that it's a timestamp, it's in UTC, and it should be an int of some form (maybe int32 is enough).

Reply:
int32 is never enough